### PR TITLE
Add New Story called Triathlon

### PR DESCRIPTION
**Pull request by alexpaul**
Please review my created story - Triathlon, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1635)

### CODE REVIEW

File: english/triathlon/triathlon.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You were always intrigued my endurance sports.
```
Comment (casey-collab) [inline english/triathlon/triathlon.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/triathlon/triathlon.md (hunk 1 above)
Comment (alexpaul) [inline english/triathlon/triathlon.md]: Not sure how to approach this. I did create a triathlon directory and then added the triathlon.mb file to it in my branch /english/triahtlon/triathlon.md

Comment (cbuckey-uda) [inline english/triathlon/triathlon.md]: @alexpaul, the point is that if a reader starts in `language.md` and keeps clicking links, they will never reach this change. You should fix this by finding a place in the existing story and adding an option that leads to your page.

**Comment by alexpaul** [comment]
@casey-collab, @cbuckey-uda I updated the subdirectory format. Please review

File: english/triathlon/english/triathlon/triathlon.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+You were always intrigued my endurance sports.
```
Comment (casey-collab) [inline english/triathlon/english/triathlon/triathlon.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@alexpaul, the problem was not with the directory structure. You need to find an existing page and add a link to it. Clicking that link should then take the reader to your new page, `triathlon.md`. That is, you should find a point in the existing story where it would make sense to sign up for a triathalon, and then add an option like "Compete in a triathalon" which leads to your page. If you can't find a good place, you'll need to get creative or change your addition.

**Comment by alexpaul** [comment]
@casey-collab, @cbuckey-uda added a link to the triathlon story hope this works out. Thanks for all the assistance

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Comment by cbuckey-uda** [comment]
Glad you figured it out!

**Pull request merged**